packages/flutter/lib/src/material/slider.dart

```diff
@@ -125,6 +125,7 @@ class Slider extends StatefulWidget {
   const Slider({
     super.key,
     required this.value,
+    this.secondaryTrackValue,
```

Comment:
It is unclear to me what a "secondary value" is. I suggest adding a paragraph explaining the secondary value, for example adapting from the issue this PR is fixing, possibly adding it to the doc of Slider or secondaryValue.

Reply:
I think I have already added information at line 209. Am I missing something here?